buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile

```diff
@@ -221,6 +221,16 @@ RUN mkdir -p /usr/lib/jvm/adoptojdk-java-12 \
   && mv bootjdk12/* /usr/lib/jvm/adoptojdk-java-12 \
   && rm -rf bootjdk12
 
+# Setup boot JDK for building Java 14 and higher
```

Comment:
It doesn't need to be changed but note that "and higher" doesn't really work because once Java 14 is available, it will be required for Java 15. Although that give a couple of months leeway between the Java 14 branch split and GA.